### PR TITLE
103753

### PR DESCRIPTION
**Pull request by jennyesoo**
Add AvDetail in product

### CODE REVIEW

File: HP.Pulsar.Search.Keyword/DataReader/ProductReader.cs [hunk 1]
```diff
@@ -297,6 +298,20 @@ private string GetCurrentROMText()
         return "Select id,currentROM,currentWebROM From ProductVersion";
     }
 
+    private string GetAvDetailText()
+    {
+        return @"
+SELECT DISTINCT p.ID AS ProductVersionId,
```
Comment (wooly905) [inline HP.Pulsar.Search.Keyword/DataReader/ProductReader.cs:304]: AS ProductId 

As I mentioned before, don't be fooled by column name.

File: HP.Pulsar.Search.Keyword/DataReader/ProductReader.cs [hunk 2]
```diff
@@ -297,6 +298,20 @@ private string GetCurrentROMText()
...
+    A.AvNo
+FROM productversion p
+LEFT JOIN Product_Brand PB ON PB.productVersionID = p.ID
+LEFT JOIN Series s WITH (NOLOCK) ON s.ProductBrandId = PB.Id
```
Comment (wooly905) [inline HP.Pulsar.Search.Keyword/DataReader/ProductReader.cs:308]: 需要這一行嗎 ?  Try without this join.
Let you know the result and how much time it takes to run this function.

**Review by wooly905 — APPROVED** [review]
Resolve the comments and then merge.

File: HP.Pulsar.Search.Keyword/DataReader/ProductReader.cs (hunk 2 above)
Comment (jennyesoo) [reply HP.Pulsar.Search.Keyword/DataReader/ProductReader.cs]: This line was not needed, I removed it.
this function runs for about 3 seconds.

**Review by wooly905 — APPROVED** [review]

**Pull request merged**